src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC } from 'react';
+import { PreparedTodo } from '../../types';
+import { Todo } from '../Todo/Todo';
+
+interface Props {
+  todosList: PreparedTodo[]
+}
+
+export const TodoList: FC<Props> = ({ todosList }) => (
```

Comment:
there is no need for this component, move the code to Todos and rename it to TodoList